charts/portworx/templates/_helpers.tpl

```diff
@@ -22,7 +22,12 @@ release: {{ .Release.Name | quote }}
 {{- end -}}
 
 {{- define "px.kubernetesVersion" -}}
-{{$version := .Capabilities.KubeVersion.GitVersion | regexFind "^v\\d+\\.\\d+\\.\\d+"}}{{$version}}
```

Comment:
This function is being used for determining pvc-controller as well. So we might want to add a new function just for storkKubernetesVersion

Reply:
updated